pwa/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: false,
+    trailingComma: 'all',
+    singleQuote: true,
+    printWidth: 120,
```

Comment:
Is this allowed characters on a single line?
If so, this is large in my opinion, as I know python listing standards are between 70-90.